lib/chat/rooms.ex

```diff
@@ -7,9 +7,20 @@ defmodule Chat.Rooms do
   alias Chat.Rooms.Registry
   alias Chat.Rooms.Room
   alias Chat.Rooms.RoomMessages
+  alias Chat.Sync.CargoRoom
 
   @doc "Returns new room {Identity, Room}"
-  def add(me, name, type \\ :public) do
+  def add(me, name, type \\ :public)
+
+  def add(me, name, :cargo) do
+    {_room_identity, %Room{} = room} = result = add(me, name, :public)
+
+    CargoRoom.activate(room.pub_key)
+
```

Comment:
This makes hard coupling

I would make UI handling to choose that to do, not the room to know about Cargo sync